akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -39,7 +39,15 @@ case object RecoveryCompleted extends RecoveryCompleted {
 @SerialVersionUID(1L)
 final case class Recover(fromSnapshot: SnapshotSelectionCriteria = SnapshotSelectionCriteria.Latest, toSequenceNr: Long = Long.MaxValue, replayMax: Long = Long.MaxValue)
 
+/** Convenience for skipping recovery of [[PersistentActor]] instances. */
+object NoRecovery { // TODO do we want this?
```

Comment:
@patriknw I thought it might be useful, as `Recover(toSequenceNr = 0L)` is a bit sneaky, maybe doesn't matter though?